rs/boundary_node/salt_sharing/canister/storage.rs

```diff
@@ -0,0 +1,32 @@
+use ic_stable_structures::{
+    memory_manager::{MemoryId, MemoryManager, VirtualMemory},
+    DefaultMemoryImpl, StableBTreeMap,
+};
+use std::cell::RefCell;
+
+pub type Timestamp = u64;
+pub const SALT_SIZE: usize = 64;
+
+type Memory = VirtualMemory<DefaultMemoryImpl>;
+pub type StableMap<K, V> = StableBTreeMap<K, V, Memory>;
+
+const MEMORY_ID_SALT: MemoryId = MemoryId::new(0);
+const MEMORY_ID_LAST_SALT_NS: MemoryId = MemoryId::new(1);
+
+thread_local! {
+    static MEMORY_MANAGER: RefCell<MemoryManager<DefaultMemoryImpl>> =
+        RefCell::new(MemoryManager::init(DefaultMemoryImpl::default()));
```

Comment:
I suppose it isn't a big deal, but let me point out that with this setup, the salt and the timestamp will be stored in an 8MiB stable memory region each (16MiB in total) which is quite wasteful.

Reply:
Good to know. I can store these two variables in one value as a tuple. One usually performs write/read operations on these two variable at the same time. So we can group them, i guess.